### PR TITLE
skipper: Fix formatting of flags

### PR DESCRIPTION
**Pull request by mikkeloscar**
The formatting of the skipper flags causes an issue in EKS clusters. Simplify by using `'` around the whole flag value and only use `"` for the eskip notation.

### CODE REVIEW

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -316,7 +316,7 @@ spec:
 {{ end }}
 {{ if or (eq .Cluster.ConfigItems.nlb_switch "pre") (eq .Cluster.ConfigItems.nlb_switch "exec") }}
           - "-forwarded-headers=X-Forwarded-For,X-Forwarded-Proto=https,X-Forwarded-Port=443"
-          - "-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}"
+          - '-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}'
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml:319]: This change is not necessary as CIDRs do not contain any quotes

Comment (mikkeloscar) [reply cluster/manifests/skipper/deployment.yaml:319]: Do you prefer not to have it?

Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/deployment.yaml:319]: There is no difference at the end so we can keep it.

**Pull request merged**